health/notifications/alarm-notify.sh.in

```diff
@@ -1859,36 +1859,40 @@ send_syslog() {
     if [ ${prefix} != ${temp1} ]; then
       if (echo ${temp1} | grep -q '@'); then
         temp2=$(echo ${temp1} | cut -d '@' -f 1)
-        host=$(echo ${temp1} | cut -d '@' -f 2)
+        server=$(echo ${temp1} | cut -d '@' -f 2)
 
-        if [ ${temp2} != ${host} ]; then
+        if [ ${temp2} != ${server} ]; then
           priority=${temp2}
         fi
 
-        port=$(echo ${host} | rev | cut -d ':' -f 1 | rev)
+        port=$(echo ${server} | rev | cut -d ':' -f 1 | rev)
 
-        if (echo ${host} | grep -E -q '\[.*\]'); then
+        if (echo ${server} | grep -E -q '\[.*\]'); then
           if (echo ${port} | grep -q ']'); then
             port=''
           else
-            host=$(echo ${host} | rev | cut -d ':' -f 2- | rev)
+            server=$(echo ${server} | rev | cut -d ':' -f 2- | rev)
           fi
         else
-          if [ ${port} = ${host} ]; then
+          if [ ${port} = ${server} ]; then
             port=''
           else
-            host=$(echo ${host} | cut -d ':' -f 1)
+            server=$(echo ${server} | cut -d ':' -f 1)
           fi
         fi
       else
         priority=${temp1}
       fi
     fi
 
-    message="${prefix} ${status} on ${host} at ${date}: ${chart} ${value_string}"
+    if [ -z ${server} ] ; then
+      server="${args_host}"
+    fi
+
+    message="${prefix} ${status} on ${server} at ${date}: ${chart} ${value_string}"
```

Comment:
```suggestion
    message="${prefix} ${status} on ${host} at ${date}: ${chart} ${value_string}"
```

Reply:
`server` is a destination we want to send log message to, right?

Reply:
good catch, thank you ! I changed all variables, but this should be `host`.